crates/isograph_lang_parser/src/token_kind.rs

```diff
@@ -75,6 +75,9 @@ pub enum IsographLangTokenKind {
     // Comments
     // #[regex("#[^\n\r]*")]
     // SingleLineComment,
+    #[regex("//")]
```

Comment:
you should be able to use `#[token("//")]`, which presumably is faster than using a regex. Source: https://github.com/maciejhirsz/logos

Reply:
Ah, my bad, thanks! Was testing which is better: show a token error of the whole "commented" line

Reply:
@rbalicki2 done!